homeassistant/components/template/vacuum.py

```diff
@@ -197,6 +197,8 @@ def __init__(
         if self._battery_level_template:
             self._attr_supported_features |= VacuumEntityFeature.BATTERY
 
+        self._attr_supported_features |= VacuumEntityFeature.STATUS
```

Comment:
`VacuumEntityFeature.STATUS` is deprecated and was only ever meant to be used by the deprecated `VacuumEntity` base class, not by the `StateVacuumEntity` base class.